dev-env/dev_env_vars.yml

```diff
@@ -14,7 +14,7 @@ kive_venv: /opt/venv_kive
 kive_db_name: kive
 kive_db_user: kive
 kive_db_host: head
-kive_media_root: /data/kive/media_root
+kive_media_root: /tmp/kive/media_root
```

Comment:
Will the media root stay on this path once setup is finished? If so, I'd rather not have it under `/tmp`. I think of `/tmp` as files that can be purged without harming the installation, the media root is definitely not that. I think we had it under `/data/kive` to mirror production. `/var/kive` would be another option if you don't like `/data` for some reason.